### ISSUE TITLE
Deploy failing, some call needs an authorization code

### ISSUE THREAD

**Issue by adamread** [google/cwv_in_ga4#3]
Running the deploy script results in the following error:

> details = "Failed to find a valid credential. The request to create a transfer config is supposed to contain an authorization code."

The trace looked like this: 
```
 File "/home/adamread/cwv_in_ga4/./deploy.py", line 579, in <module>
    main()
  File "/home/adamread/cwv_in_ga4/./deploy.py", line 518, in main
    deploy_scheduled_materialize_query(project_id, args.region, args.ga_property)
  File "/home/adamread/cwv_in_ga4/./deploy.py", line 236, in deploy_scheduled_materialize_query
    transfer_config = transfer_client.create_transfer_config(
  File "/home/adamread/.local/lib/python3.9/site-packages/google/cloud/bigquery_datatransfer_v1/services/data_transfer_service/client.py", line 757, in create_transfer_config
    response = rpc(request, retry=retry, timeout=timeout, metadata=metadata,)
  File "/home/adamread/.local/lib/python3.9/site-packages/google/api_core/gapic_v1/method.py", line 154, in __call__
    return wrapped_func(*args, **kwargs)
  File "/home/adamread/.local/lib/python3.9/site-packages/google/api_core/grpc_helpers.py", line 68, in error_remapped_callable
    raise exceptions.from_grpc_error(exc) from exc
google.api_core.exceptions.InvalidArgument: 400 Failed to find a valid credential. The request to create a transfer config is supposed to contain an authorization code.
```

**Comment by adamread** [comment]
Looks like you need to run `gcloud auth login` before running the deployment script. Trying to verify.

**Comment by adamread** [comment]
That was not the case, at least not when using the cloud console.

The failing call is bigquery_datatransfer.DataTransferServiceClient().create_transfer_config()

The error says we need a credential, but the [docs](https://cloud.google.com/python/docs/reference/bigquerydatatransfer/latest/google.cloud.bigquery_datatransfer_v1.types.CreateTransferConfigRequest#attributes) say it's optional. Not sure why everything else works with the credentials we already have, but we can try following the docs to see if it helps.

**Comment by adamread** [comment]
Fixed by PR#12
